src/components/SignerPanel/ActionPathsContent.js

```diff
@@ -239,6 +243,15 @@ class ActionPathsContent extends React.Component {
             </Info.Action>
           </div>
         )}
+        {external && (
+          <div css="margin-top: 20px">
+            <Info.Alert title="Warning">
+              {`Be aware that this is an attempt to execute a transaction on another app that is ${
+                installed ? '' : 'not'
+              } installed in this organization. You may want to double check that app's functionality before proceeding.`}
+            </Info.Alert>
```

Comment:
I think the intent was to keep the original text for non-installed external contracts:

```
Be aware that this is an attempt to execute a transaction on an *external contract* that has not been reviewed or audited. This means that it might behave unexpectedly. Please *be sure your trust this contract* before proceeding.
```

But for installed apps:

```
Be aware that this is an attempt to execute a transaction on another app that is installed in this organization. You may want to double check that app's functionality before proceeding.
```